admin_guide/high_availability.adoc

```diff
@@ -10,50 +10,54 @@
 toc::[]
 
 == Overview
-This document describes how to setup high available services on your
+This document describes how to set up highly-available services on your
 OpenShift cluster.
 
 == IP Failover
-This section describes how to make any IP based network service highly available using virtual IP addresses and IP failover.
+The Kubernetes replication controller ensures that the deployment requirements,
+in particular the number of replicas, are satisfied when the appropriate
+resources are available. The
+link:../architecture/core_objects/routing.html[router] can be run with two or
+more replicas to provide a highly-available service that is resilient to router
+failures. Depending on how the router instances are discovered (via a service,
+DNS entry, or IP addresses), this could impose operational requirements to
+handle failure cases when one or more router instances are "unreachable".
+
+For some IP-based traffic services, IP addresses should always be serviced for
```

Comment:
@ramr Edited so that in the above instance it says "virtual IP addresses". However, is this the only instance that needed changing to VIPs? I'd hate to confused the two.